### PR TITLE
M2-1685

### PR DESCRIPTION
**Pull request by BamMironov**
Added custom `ImageBackground` to resolve: https://mindlogger.atlassian.net/browse/M2-1685

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
     "ios:production-release": "react-native run-ios --scheme MindloggerMobile --configuration Release"
   },
   "dependencies": {
-    "@georstat/react-native-image-cache": "^2.3.0",
+    "@georstat/react-native-image-cache": "^3.1.0",
     "@hookform/resolvers": "^2.9.10",
     "@miblanchard/react-native-slider": "^2.3.1",
```
Comment (anq83) [inline package.json:39]: Please, notify to Natalia of possible regressions related to changed library versions

File: src/shared/ui/ImageBackground.tsx [hunk 2]
```diff
@@ -1,60 +1,31 @@
...
-        resizeMode="cover"
-        sourceAnimationDuration={500}
-        onLoadEnd={loadingFinished}
-        loadingImageComponent={() => (
-          <ActivityIndicator size="large" color="$secondary" />
-        )}
-        onError={onError}
-      />
-
-      {loader && isLoading && loader}
```
Comment (anq83) [inline src/shared/ui/ImageBackground.tsx:51]: Don't we need loader anymore?

File: yarn.lock [hunk 3]
```diff
@@ -7296,7 +7288,7 @@ react-native-device-info@^10.6.0:
   resolved "https://registry.yarnpkg.com/react-native-device-info/-/react-native-device-info-10.6.0.tgz#51f1b2ec98abc32747149b0a5b7fb662b44d50e4"
   integrity sha512-/MmINdojWdw2/9rwYpH/dX+1gFP0o78p8yYPjwxiPhoySSL2rZaNi+Mq9VwC+zFi/yQmJUvHntkKSw2KUc7rFw==
 
-react-native-file-access@^2.4.4, react-native-file-access@^2.5.2:
+react-native-file-access@^2.5.2:
```
Comment (anq83) [inline yarn.lock:7291]: Please, notify to Natalia of possible regressions related to changed library versions

**Review by anq83 — APPROVED** [review]

File: src/shared/ui/ImageBackground.tsx (hunk 2 above)
Comment (BamMironov) [reply src/shared/ui/ImageBackground.tsx:51]: No, we don't

**Pull request merged**